contracts/contracts/strategies/BridgedWOETHStrategy.sol

```diff
@@ -0,0 +1,185 @@
+            "Caller is not the Strategist or Governor"
+        );
+        _;
+    }
+
+    constructor(
+        BaseStrategyConfig memory _stratConfig,
+        address _weth,
+        address _bridgedWOETH,
+        address _oracleFeed
+    ) InitializableAbstractStrategy(_stratConfig) {
+        weth = IWETH9(_weth);
+        bridgedWOETH = IERC20(_bridgedWOETH);
+
+        oracleFeed = AggregatorV3Interface(_oracleFeed);
+        oracleFeedDecimals = AggregatorV3Interface(_oracleFeed).decimals();
+    }
+
+    function initialize() external onlyGovernor initializer {
+        InitializableAbstractStrategy._initialize(
+            new address[](0), // No reward tokens
+            new address[](0), // No assets
+            new address[](0) // No pTokens
+        );
+    }
+
+    function getBridgedWOETHValue(uint256 amount)
+        public
+        view
+        returns (uint256 oethValue)
+    {
+        // slither-disable-next-line unused-return
+        (, int256 oraclePrice, , uint256 updatedAt, ) = AggregatorV3Interface(
+            oracleFeed
+        ).latestRoundData();
+
+        require(
+            updatedAt + MAX_PRICE_STALENESS >= block.timestamp,
+            "Oracle price too old"
+        );
+
+        // Price per unit bridged wOETH
+        oethValue = uint256(oraclePrice).scaleBy(18, oracleFeedDecimals);
+
+        // Price for `amount`
+        oethValue = (amount * oethValue) / 1 ether;
+    }
+
+    // TODO: Add minAmount
+    function depositBridgedWOETH(uint256 woethAmount)
+        external
+        onlyGovernorOrStrategist
+    {
+        // Transfer in all bridged wOETH tokens
+        bridgedWOETH.transferFrom(msg.sender, address(this), woethAmount);
+
+        // Figure out how much they are worth
+        uint256 oethToMint = getBridgedWOETHValue(woethAmount);
+
+        // There's no pToken, however, it just uses WOETH address in the event
+        emit Deposit(address(weth), address(bridgedWOETH), oethToMint);
+
+        // Mint OETHb tokens and transfer it to the caller
+        IVault(vaultAddress).mintToForStrategy(msg.sender, oethToMint);
+    }
+
+    function withdrawBridgedWOETH(uint256 woethAmount, address receiver)
+        external
+        onlyGovernorOrStrategist
+    {
+        uint256 oethToBurn = getBridgedWOETHValue(woethAmount);
+
+        // There's no pToken, however, it just uses WOETH address in the event
+        emit Withdrawal(address(weth), address(bridgedWOETH), oethToBurn);
+
+        // Burn OETHb
+        IVault(vaultAddress).burnFromForStrategy(msg.sender, oethToBurn);
+
+        // Transfer WOETH back
+        bridgedWOETH.transfer(receiver, woethAmount);
+    }
+
+    /**
+     * @notice Returns the amount of backing WETH the strategy holds
+     * @param _asset      Address of the asset
+     * @return balance    Total value of the asset in the platform
+     */
+    function checkBalance(address _asset)
```

Comment:
The deposit and the withdraw functions would require bad behavior by both oracle AND the strategist to have a problem. This is okay.

The checkBalance problem would have a problem if there was a problem with either mainnet woeth OETH with the oracle. This is worth some thinking about.

In the past, we not had a checkBalance function that had explicit reverts. `getBridgedWOETHValue` will revert if the updated date is wrong. (This is probably required here). I wonder if we should also revert if things are out of range in some way.